connectors/api/Makefile

```diff
@@ -0,0 +1,44 @@
+ROOT_DIR := ../..
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/hack/make-rules/tools.mk
+
+ABS_ROOT_DIR := $(abspath ${ROOT_DIR})
+OPENAPI_GENERATOR_IMG := openapitools/openapi-generator-cli:v6.2.0
```

Comment:
@rohithdv can you please update hack/tools/requirements.sh with `OPENAPI_GENERATOR_VERSION` version and use it in this Makefile and site/Makfile. Also, after using this fixed version the files `pkg/connectors/policymanager/clients/policymanager_openapi.go`  and `fybrik/pkg/connectors/policymanager/openapiclient/model.go` that call the generated code can be updated so that compilation will pass.

Reply:
I have set OPENAPI_GENERATOR_VERSION to be 5.3.1 and made the suggested changes. Please review. Thanks.

Reply:
LGTM Thanks for addressing it